geopandas/io/file.py

```diff
@@ -84,6 +84,7 @@ def _import_pyogrio():
             PYOGRIO_GE_081 = Version(
                 Version(pyogrio.__version__).base_version
             ) >= Version("0.8.1")
+            PYOGRIO_GE_06 = Version(pyogrio.__version__) >= Version("0.6.0")
```

Comment:
I note this compares on base_version for PYOGRIO_GE_081, I think this is just for alphas/ betas so shouldn't matter for 0.6 since it's old.